src/pages/SplitPosition/Form.tsx

```diff
@@ -196,6 +201,25 @@ export const Form = ({ allowanceMethods, onCollateralChange, splitPosition, toke
     [12, 13, 14, 15],
   ]
 
+  const splitPositionPreview = useMemo(() => {
+    if (splitFromPosition && position) {
+      return trivialPartition(outcomeSlot).map((indexSet) => {
+        return positionString(
+          [...position.conditionIds, conditionIdToPreviewShow],
+          [...[position.indexSets], indexSet],
+          amount,
+          collateralToken
+        )
+      })
+    } else if (conditionIdToPreviewShow) {
+      return trivialPartition(outcomeSlot).map((indexSet) => {
+        return positionString([conditionIdToPreviewShow], [indexSet], amount, collateralToken)
+      })
+    } else {
+      return []
+    }
```

Comment:
Since `conditionIdToPreviewShow` is used in both cases I think we need to rethink how/what we are checking here. I would go for something like:
```typescript
if (!conditionIdToPreviewShow || (splitFromPosition && !position)) {
  return []
}

if (splitFromPosition) {
  ...
} else {
  ...
}
```